public_html/wp-content/mu-plugins/load-other-mu-plugins.php

```diff
@@ -46,9 +47,18 @@ function wcorg_include_mu_plugin_folders() {
 		if ( is_array( $plugins ) ) {
 			foreach ( $plugins as $plugin ) {
 				if ( is_file( $plugin ) ) {
-					require_once( $plugin );
+					require_once $plugin;
 				}
 			}
 		}
 	}
 }
+
+/**
+ * Load every mu-plugin that only runs on events network
+ */
+function wcorg_include_events_network_mu_plugins() {
+	if ( 'SITE_ID_CURRENT_SITE' === 2 ) {
```

Comment:
This PR should follow whatever implementation is done relative to this review: 
https://github.com/WordPress/wordcamp.org/pull/907#discussion_r1236243881

Reply:
The latest would be `if ( EVENTS_NETWORK_ID === SITE_ID_CURRENT_SITE ) {`

See https://github.com/WordPress/wordcamp.org/pull/907/commits/fa837c09dd97a97d6861f8642e9aafc25f2ac215 and https://github.com/WordPress/wordcamp.org/pull/907/commits/cad6d965f115bb5c1e4fc4832def14814971d2af